build-config/make/efivar.make

```diff
@@ -82,7 +83,11 @@ $(EFIVAR_INSTALL_STAMP): $(SYSROOT_INIT_STAMP) $(EFIVAR_BUILD_STAMP)
 		cp -av $(DEV_SYSROOT)/usr/bin/$$file $(SYSROOTDIR)/usr/bin/ ; \
 	     done
 	$(Q) for file in $(EFIVAR_LIBS); do \
-		cp -av $(DEV_SYSROOT)/usr/lib/$$file $(SYSROOTDIR)/usr/lib/ ; \
+		if [ "$(ARCH)" == "arm64" ] || [ "$(ARCH)" == "x86_64" ]; then \
+		    cp -av $(DEV_SYSROOT)/usr/lib64/$$file $(SYSROOTDIR)/usr/lib/ ; \
+		else \
+		    cp -av $(DEV_SYSROOT)/usr/lib/$$file $(SYSROOTDIR)/usr/lib/ ; \
+		fi \
```

Comment:
I don't quite understand the `if` statement.  previously this file was only for x86_64 using "lib".  this change now makes x86_64 use "lib64".  is that what you meant?

the `else` case would never be taken as only arm64 and x86_64 use efivar.